src/main/java/raisetech/student/management/validation/SexValidator.java

```diff
@@ -0,0 +1,34 @@
+package raisetech.student.management.validation;
+
+import jakarta.validation.ConstraintValidator;
+import jakarta.validation.ConstraintValidatorContext;
+
+public class SexValidator implements ConstraintValidator<ValidSex, String> {
+
+ // @Override
+  //public boolean isValid(String value, ConstraintValidatorContext context){
+    //if(value == null){
+     // return true;
+   // }
+  //  return value.equals("男性") || value.equals("女性") || value.equals("その他");
+  //}
+
+  // enumの定義をこのクラス内に含めます
+  public enum Sex {
+    男性, 女性, その他;
```

Comment:
enumも英語で定義したいですね。
```suggestion
    MALE, FEMALE, OTHERS;
```

Reply:
日本語を使いたい場合はフィールド変数を使うとよいです。
```suggestion
    MALE("男性"), FEMALE("女性"), OTHERS("その他");
```